native/mod_cluster_slotmem/sharedmem_util.c

```diff
@@ -506,14 +506,25 @@ static apr_status_t ap_slotmem_mem(ap_slotmem_t *score, int id, void**mem)
 static apr_status_t ap_slotmem_alloc(ap_slotmem_t *score, int *item_id, void **mem)
 {
     int ff;
+    int id = 0;
     int *ident;
     apr_status_t rv;
     ident = score->ident;
-    ff = ident[0];
+    if (*item_id != 0) {
+        /* we want a specific id, not a new one */
+        ff = ident[id];
+        while (ff < score->num && ff!=*item_id) {
```

Comment:
I missed those missing spaces here. But no problem, this will get addressed with the code style in the future.